### PR TITLE
feat: add tests for bulk course email task manager components

### PR DESCRIPTION
**Pull request by justinhynes**
[MICROBA-1621]

* Fix display bug for the error Alert component found during test writing
* Add tests for BulkEmailTaskHistory component
* Fix bug with a column name in the bulk course email content history table
* Add tests for the BulkEmailContentHistory component
* Add tests for the BulkEmailPendingTasks component

### CODE REVIEW

File: src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      const email = emails[0];
+      expect(await screen.findByText(email.created)).toBeTruthy();
+      expect(await screen.findByText(email.number_sent)).toBeTruthy();
+      expect(await screen.findByText(email.requester)).toBeTruthy();
+      expect(await screen.findByText(email.sent_to.join(', '))).toBeTruthy();
+      expect(await screen.findByText(email.email.subject)).toBeTruthy();
+      expect(await screen.findAllByText('View Message')).toBeTruthy();
+
+      // verify modal behavior
+      const viewMessageButton = await screen.findByText('View Message');
```
Comment (Tj-Tracy) [inline src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx:65]: we may want to extract this portion of logic out onto its own test, or change the description of this test to reflect the modal interaction. Overall I'd consider the modal its own "unit".

**Review by Tj-Tracy — APPROVED** [review]
overall LGTM 👍 one non-blocking comment about a test.

File: src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx (hunk 1 above)
Comment (justinhynes) [reply src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx:65]: Fair enough!

Comment (justinhynes) [reply src/components/bulk-email-tool/bulk-email-task-manager/test/BulkEmailContentHistory.test.jsx:76]: I moved pieces that verified the modal behavior and content into its own unit test. Thanks for the feedback! :)

**Pull request merged**